deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elastic-agent
+  namespace: kube-system
+  labels:
+    app: elastic-agent
+spec:
+  selector:
+    matchLabels:
+      app: elastic-agent
+  template:
+    metadata:
+      labels:
+        app: elastic-agent
+    spec:
+      serviceAccountName: elastic-agent
+      containers:
+        - name: elastic-agent
+          image: docker.elastic.co/beats/elastic-agent:%VERSION%
+          args: [
+            "-c", "/etc/agent.yml",
+            "-e", "-d", "composable.providers.kubernetes",
+          ]
+          env:
+            - name: ES_USERNAME
+              value: "elastic"
+            - name: ES_PASSWORD
+              value: ""
+            - name: ES_HOST
+              value: ""
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            # this is needed because we cannot use hostNetwork
```

Comment:
Why?

Reply:
If we use `hostNetwork` on Deployment too, it will cause a conflict on grpc port of Agent between the Deployment Pod and the Daemonset Pod that will end up on the same k8s node. I think that grpc port is configurable but I don't see an obvious reason of having the Deployment Pod under `hostNetwork`.

Reply:
Ah, we expect to deploy both at the same time, makes sense.